src/views/Operations/FactoryReset/FactoryReset.vue

```diff
@@ -1,140 +1,142 @@
               stacked
             >
-              <b-form-radio
+              <BFormRadio
                 class="mb-1"
                 value="resetBios"
                 aria-describedby="reset-bios"
                 :disabled="serverStatus !== 'off'"
                 data-test-id="factoryReset-radio-resetBios"
               >
                 {{ $t('pageFactoryReset.form.resetBiosOptionLabel') }}
-              </b-form-radio>
-              <b-form-text id="reset-bios" class="ml-4 mb-3">
+              </BFormRadio>
+              <label id="reset-bios">
                 {{ $t('pageFactoryReset.form.resetBiosOptionHelperText') }}
-              </b-form-text>
+              </label>
 
-              <b-form-radio
+              <BFormRadio
                 class="mb-1"
                 value="resetToDefaults"
                 aria-describedby="reset-to-defaults"
                 data-test-id="factoryReset-radio-resetToDefaults"
                 :disabled="serverStatus !== 'off'"
               >
                 {{ $t('pageFactoryReset.form.resetToDefaultsOptionLabel') }}
-              </b-form-radio>
-              <b-form-text id="reset-to-defaults" class="ml-4 mb-3">
+              </BFormRadio>
+              <label id="reset-to-defaults">
                 {{
                   $t('pageFactoryReset.form.resetToDefaultsOptionHelperText')
                 }}
-              </b-form-text>
-            </b-form-radio-group>
-          </b-form-group>
-          <b-button
+              </label>
+            </BFormRadioGroup>
+          </BFormGroup>
+          <BButton
+            v-b-modal.modal-reset
             type="submit"
             variant="primary"
             :disabled="serverStatus !== 'off'"
             data-test-id="factoryReset-button-submit"
           >
             {{ $t('global.action.reset') }}
-          </b-button>
-        </b-col>
-      </b-row>
-    </b-form>
+          </BButton>
+        </BCol>
+      </BRow>
+    </BForm>
 
     <!-- Modals -->
     <modal-reset :reset-type="resetOption" @okConfirm="onOkConfirm" />
-  </b-container>
+  </BContainer>
 </template>
 
-<script>
-import Alert from '@/components/Global/Alert';
-import PageTitle from '@/components/Global/PageTitle';
-import BVToastMixin from '@/components/Mixins/BVToastMixin';
-import LoadingBarMixin from '@/components/Mixins/LoadingBarMixin';
-import ModalReset from './FactoryResetModal';
+<script setup>
+import { ref, onMounted, computed } from 'vue';
+import Alert from '@/components/Global/Alert.vue';
+import PageTitle from '@/components/Global/PageTitle.vue';
+import ModalReset from './FactoryResetModal.vue';
+import useLoadingBar from '@/components/Composables/useLoadingBarComposable';
+import useToastComposable from '@/components/Composables/useToastComposable';
+import { FactoryResetStore, AuthenticationStore, GlobalStore } from '@/store';
+import eventBus from '@/eventBus';
+
+const Global = GlobalStore();
```

Comment:
let the variable names start with lowercase: "globalStore", "authenticationStore", "factoryResetStore", "toast"